tests/utilities/test_template_formatters.py

```diff
@@ -0,0 +1,125 @@
+from typing import Any, Dict, Type
+
+import pytest
+
+from phoenix.utilities.template_formatters import (
+    FStringTemplateFormatter,
+    MustacheTemplateFormatter,
+    TemplateFormatter,
+)
+
+
+@pytest.mark.parametrize(
+    "formatter_cls, template, variables, expected_prompt",
+    (
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{ hello }}",
+            {"hello": "world"},
+            "world",
+            id="mustache-whitespace-both-sides",
+        ),
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{hello}}",
+            {"hello": "world"},
+            "world",
+            id="mustache-no-whitespace",
+        ),
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{  hello}}",
+            {"hello": "world"},
+            "world",
+            id="mustache-whitespace-left",
+        ),
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{hello }}",
+            {"hello": "world"},
+            "world",
+            id="mustache-whitespace-right",
+        ),
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{ hello }}, {{ world }}",
+            {"hello": "1", "world": "2"},
+            "1, 2",
+            id="mustache-multiple-variables",
+        ),
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{ hello }} + {{hello}} = {{ world }}",
+            {"hello": "1", "world": "2"},
+            "1 + 1 = 2",
+            id="mustache-duplicate-variables",
+        ),
+        pytest.param(
+            MustacheTemplateFormatter,
+            "{{ hello }}, {{ world }}",
```

Comment:
Does the mustache formatter support escaping?

I have it setup in the frontend to escape `\{{variable}}`, removing the slash during format

Reply:
Good call, will add.